src/pkg/projects/Microsoft.NETCore.DotNetHost/host.wxs

```diff
@@ -8,16 +8,22 @@
     <ComponentGroup Id="InstallSharedHostandDetectionKeys">
 
       <Component Id="cmpCoreHost" Directory="DOTNETHOME" Guid="{45399BBB-DDA5-4386-A2E9-618FB3C54A18}" >
-        <File Id="fileCoreHostExe" KeyPath="yes" Source="$(var.HostSrc)\dotnet.exe" />
+        <File Id="fileCoreHostExe" KeyPath="yes" Source="$(var.HostSrc)\dotnet.exe">
+          <CopyFile Id="copyFileCoreHostExe" DestinationProperty="$(var.Program_Files)dotnet" />
```

Comment:
I don't see how `DestinationProperty="$(var.Program_Files)dotnet"` works... `Program_Files` is `ProgramFilesFolder` (or 64 bit) at this point, so `DestinationProperty` would be e.g. `ProgramFilesFolderdotnet`. Then it's evaluated as a property at install-time?

Reply:
I see `bundle.wxs` doing something similar:

```xml
<Variable Name="DOTNETHOME" Type="string" Value="[$(var.Program_Files)]dotnet" bal:Overridable="no" />
```

The `[]` makes that one make sense to me.

Reply:
The $(var) should resolve to [ProgramFilesFolder] (with the square brackets). That in turn should resolve to the built-in property which should have a trailing slash AFAIK, but it would be good to confirm.

Reply:
This is the current way the var is defined, the same in the wxi files I found in Core-SDK and ASP.NET Core:

https://github.com/dotnet/core-setup/blob/0e6effe229e0913dcd71bc20d63faac8cd34391b/src/pkg/packaging-tools/windows/variables.wxi#L17

Reply:
Depending on the trailing slash seems safe based on this, assuming the rest goes right: https://stackoverflow.com/q/2733628.

Reply:
It appears from the log that the property has a trailing slash:
MSI (s) (2C:68) [15:23:57:956]: Dir (target): Key: ProgramFiles64Folder	, Object: C:\Program Files\

Reply:
Oh, wait I see what's going on. So the variable is used to resolve the <Directory> However, for Copy file you need to provide the actual folder.

So you want something like DestinationProperty="[ProgramFiles]dotnet". The $(var) reference is a preprocessor variable so it will literally resolve the authoring to DestinationProperty="ProgramFilesdotnet"

Reply:
Fixed.